src/main/play-doc/operation/Install.md

```diff
@@ -41,7 +41,7 @@ echo "JAVA_HOME=/usr/lib/jvm/java-8-oracle" | sudo tee -a /etc/environment
 
 ## Installing ConductR on the first machine
 
-This tutorial uses three systems with the addresses `172.17.0.{1,2,3}`. To simplify the installation and configuration instructions, we are going to use the hostname command. Please ensure the hostname is set correctly or substitute your addresses as appropriate for $(hostname). To set the hostname, pass the ip address to hostname.
+This tutorial uses three systems with the addresses `172.17.0.{1,2,3}` for ConductR and `172.17.0.{11,12,13}` for ConductR Agent. To simplify the installation and configuration instructions, we are going to use the hostname command. Please ensure the hostname is set correctly or substitute your addresses as appropriate for $(hostname). To set the hostname, pass the ip address to hostname.
```

Comment:
The agents should be in a different sub-net, e.g. 172.18.0.{11,12,13}. This is to support a DMZ execution model.